code/game/gamemodes/game_mode.dm

```diff
@@ -280,19 +281,19 @@
 	var/intercepttext = "<b><i>Central Command Status Summary</i></b><hr>"
 	intercepttext += "<b>Central Command has intercepted and partially decoded a Syndicate transmission with vital information regarding their movements. The following report outlines the most \
 	likely threats to appear in your sector.</b>"
-	var/list/possible_modes = list()
-	possible_modes.Add("blob", "changeling", "clock_cult", "cult", "extended", "gang", "malf", "nuclear", "revolution", "traitor", "wizard")
-	possible_modes -= name //remove the current gamemode to prevent it from being randomly deleted, it will be readded later
-
-	for(var/i in 1 to 6) //Remove a few modes to leave four
-		possible_modes -= pick(possible_modes)
-
-	possible_modes |= name //Re-add the actual gamemode - the intercept will thus always have the correct mode in its list
-	possible_modes = shuffle(possible_modes) //Meta prevention
-
-	var/datum/intercept_text/i_text = new /datum/intercept_text
-	for(var/V in possible_modes)
-		intercepttext += i_text.build(V)
+	var/list/report_weights = config.mode_false_report_weight.Copy()
+	report_weights[config_tag] = 0 //Prevent the current mode from being falsely selected.
+	var/list/reports = list()
+	for(var/i in 1 to rand(3,5)) //Between three and five wrong entries on the list.

```

Comment:
can you do this without changing functionality or report results just once?

Reply:
If by changing functionality, you mean have a consistent number of wrong reports, rather than a random number, sure that's possible.

But what do you mean by report results just once?  We need to select multiple random false reports, and we need to ensure there aren't duplicate false reports.

Reply:
As in, not adding reports for disabled modes.

Reply:
I see a problem with that then.

If an admin selects a disabled mode, and it shows up in the list, then the captain will know for sure that it's monkey roundtype.

Which is why I've kept disabled modes in, but with very low weight for them being included in roundstart reports.

If you think the weights need to be adjusted, that's easily doable.

Reply:
Also, it DID add reports for disabled modes before.... like malf.